src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -160,6 +166,38 @@ object DepositHandler {
     }
   }
 
+  private def resolveFetchItems(bagitDir: File)(implicit id: String): Try[Unit] = Try {
+    log.debug(s"[$id] Checking fetch.txt")
```

Comment:
* So you intended to say: "Checking if bag contains fetch.txt" or something. 
* Failing fast has the advantage that it becomes much easier to find the cause of an error.